### ISSUE TITLE
Add call to setAdWillAutoplay

### ISSUE THREAD

**Issue by shawnbuso** [googleads/videojs-ima#484]

**Comment by shawnbuso** [comment]
Blocked by #341

**Comment by Smartclip-ES** [comment]
Hi there!

My video-js + IMA implementation was working fine until a few days ago. Now I can´t mute the ad on some VPAIDs.
As you can see in the link below, this implementation is quite different to your example.

https://em2.storage.oraclecloud.com/v1/Storage-a482323/smartclip-services/sc_player/simple.html?vast=https%3A%2F%2Fdev.smartclip.net%2Fusr%2Feme%2FtestVast%2Fcoke.xml&autoplay=true&muted=true&poster=https://s.libertaddigital.com/videos/los-favoritos-de-la-temporada-del-motor-6060167-1.jpg&content_video=https://em2.storage.oraclecloud.com/v1/Storage-a482323/smartclip-services/sc_player/dependencies/media/sc_background.mp4&height=611.7018750000027&width=1087.47&container=sc_player14050323


You will see that i simply create the video player as muted and init the video if the var autoplay is set to true.

Is there any simple way to insert the changes needed to make it work without copying your full example?

Thanks in advance

**Comment by shawnbuso** [comment]
The example we created is designed to work cross-platform - some platforms allow you to autoplay anything any time, some have limited support, and others never let you autoplay. The example we have will autoplay if allowed, and fall back to click-to-play if not. That makes it a bit complicated.

All that aside, it shouldn't be affecting your ability to mute a VPAID ad. I tested your page, but in my tests I was able to mute the ad that loaded. Are you only seeing this on specific devices or platforms?
